diff -aurN dhcpcd-6.1.0.orig/dhcpcd.conf dhcpcd-6.1.0/dhcpcd.conf
--- dhcpcd-6.1.0.orig/dhcpcd.conf	2013-10-15 11:32:22.552282406 +0200
+++ dhcpcd-6.1.0/dhcpcd.conf	2013-10-15 11:33:07.360003940 +0200
@@ -5,10 +5,10 @@
 hostname
 
 # Use the hardware address of the interface for the Client ID.
-#clientid
+clientid
 # or
 # Use the same DUID + IAID as set in DHCPv6 for DHCPv4 ClientID as per RFC4361.
-duid
+#duid
 
 # Persist interface configuration when dhcpcd exits.
 persistent
